tests/bootstrap.php

```diff
@@ -58,29 +58,33 @@
         'plugins' => [TEST_APP . 'Plugin' . DS],
         'templates' => [APP . 'Template' . DS],
         'locales' => [APP . 'Locale' . DS],
-    ]
+    ],
 ]);
 
 Configure::write('Session', [
-    'defaults' => 'php'
+    'defaults' => 'php',
 ]);
 
 Cache::config([
     '_cake_core_' => [
         'engine' => 'File',
         'prefix' => 'cake_core_',
-        'serialize' => true
+        'serialize' => true,
     ],
     '_cake_model_' => [
         'engine' => 'File',
         'prefix' => 'cake_model_',
-        'serialize' => true
+        'serialize' => true,
     ],
     'default' => [
         'engine' => 'File',
         'prefix' => 'default_',
-        'serialize' => true
-    ]
+        'serialize' => true,
+    ],
+]);
+
+Configure::write('Error', [
+    'errorLevel' => E_ALL & ~E_USER_DEPRECATED,
```

Comment:
Why is this needed?

Reply:
"broken" Buildchain: https://travis-ci.org/github/Oefenweb/cakephp-vat-number-check/jobs/680926238

Reply:
reverted this Commit

Reply:
I see. It shouldn't be hard to fix these errors

Reply:
See https://github.com/Oefenweb/cakephp-vat-number-check-webservice/blob/3.x/tests/bootstrap.php

Reply:
> It shouldn't be hard to fix these errors

It was a bit hard and a lot of work, but everything is nice a shiny again :-)